Dockerfile

```diff
@@ -2,32 +2,31 @@
 # > docker build -t gaia .
 # > docker run -it -p 46657:46657 -p 46656:46656 -v ~/.wasmd:/root/.wasmd -v ~/.wasmcli:/root/.wasmcli gaia wasmd init
 # > docker run -it -p 46657:46657 -p 46656:46656 -v ~/.wasmd:/root/.wasmd -v ~/.wasmcli:/root/.wasmcli gaia wasmd start
-FROM golang:alpine AS build-env
+FROM golang:1.13-buster AS build-env
```

Comment:
The comments above are outdated. Proper ones are in the README. let's remove them here.

Reply:
Yes, let's remove all docs except readme